bundles/org.openhab.persistence.influxdb/pom.xml

```diff
@@ -16,82 +16,87 @@
 
   <properties>
     <bnd.importpackage>
-      !javax.annotation;!android.*,!com.android.*,!com.google.appengine.*,!dalvik.system,!kotlin.*,!kotlinx.*,!org.conscrypt,!sun.security.ssl,!org.apache.harmony.*,!org.apache.http.*,!rx.*,!org.msgpack.*
+      !javax.annotation.*;!android.*,!com.android.*,!com.google.appengine.*,!dalvik.system,!kotlin.*,!kotlinx.*,!org.conscrypt,!sun.security.ssl,!org.apache.harmony.*,!org.apache.http.*,!rx.*,!org.msgpack.*
     </bnd.importpackage>
+    <okhttp3.version>3.14.9</okhttp3.version>
+    <retrofit.version>2.7.2</retrofit.version>
+    <influx2.version>1.15.0</influx2.version>
+    <influx1.version>2.21</influx1.version>
   </properties>
 
   <dependencies>
     <!-- START InfluxDB 2.0 -->
-    <!-- START influxdb-client-java -->
     <dependency>
       <groupId>com.influxdb</groupId>
       <artifactId>influxdb-client-java</artifactId>
-      <version>1.6.0</version>
+      <version>${influx2.version}</version>
     </dependency>
     <dependency>
+      <groupId>com.influxdb</groupId>
       <artifactId>influxdb-client-core</artifactId>
+      <version>${influx2.version}</version>
+    </dependency>
+    <dependency>
       <groupId>com.influxdb</groupId>
-      <version>1.6.0</version>
+      <artifactId>flux-dsl</artifactId>
+      <version>${influx2.version}</version>
     </dependency>
+
     <dependency>
-      <artifactId>converter-gson</artifactId>
       <groupId>com.squareup.retrofit2</groupId>
-      <version>2.5.0</version>
+      <artifactId>converter-gson</artifactId>
+      <version>${retrofit.version}</version>
     </dependency>
     <dependency>
+      <groupId>com.squareup.retrofit2</groupId>
       <artifactId>converter-scalars</artifactId>
+      <version>${retrofit.version}</version>
+    </dependency>
+    <dependency>
+      <artifactId>retrofit</artifactId>
       <groupId>com.squareup.retrofit2</groupId>
-      <version>2.5.0</version>
+      <version>${retrofit.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.squareup.okhttp3</groupId>
+      <artifactId>okhttp</artifactId>
+      <version>${okhttp3.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.squareup.okhttp3</groupId>
+      <artifactId>logging-interceptor</artifactId>
+      <version>${okhttp3.version}</version>
     </dependency>
     <dependency>
-      <artifactId>gson</artifactId>
       <groupId>com.google.code.gson</groupId>
-      <version>2.9.1</version>
+      <artifactId>gson</artifactId>
+      <version>2.8.5</version>
```

Comment:
(why) Is this downgrade needed? (Note: version is currently 2.9.1 in core) It was last upgraded with #14090, and also prior to that #12772 because of security fix.